modules/dragster/src/test/ts/atomic/datatransfer/DataTransferCloneTest.ts

```diff
@@ -0,0 +1,92 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { KAssert } from '@ephox/katamari-assertions';
+import { PlatformDetection } from '@ephox/sand';
+import { assert } from 'chai';
+
+import { cloneDataTransfer, createDataTransfer, getDragImage } from 'ephox/dragster/datatransfer/DataTransfer';
+import { Event, getEvent, setDragstartEvent } from 'ephox/dragster/datatransfer/Event';
+import { getMode, Mode, setMode, setReadOnlyMode, setReadWriteMode } from 'ephox/dragster/datatransfer/Mode';
+
+describe('atomic.dragster.datatransfer.DataTransferCloneTest', () => {
+  const dragImage = {
```

Comment:
If you expose the DragImage type then that can be used in this test so anytime we add some new property, or list references you would benefit from having a proper type on it.